src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,72 @@
+// Retrieved and modified from https://usehooks-ts.com/react-hook/use-local-storage
+
+import { useCallback, useEffect, useState } from 'react'
+import { LOCAL_STORAGE_EVENT } from '~/constants/localStorage'
+
+export const useLocalStorage = <T>(
+  key: string | null,
```

Comment:
Why do we allow `null` here?

Reply:
no clue, must have been a formsg specific thing :P, removing.